File: GOVI-Front/src/app/model/Quai.ts
import { Retournement } from './Retournement';

export class Quai {
  id: string;
  retournements: Retournement[];
  constructor(id: string, retournements: Retournement[]) {
    this.id = id;
    this.retournements = retournements;
  }
}

File: GOVI-Back/src/main/java/com/sncf/govi/service/model/Quai.java
package com.sncf.govi.service.model;

import lombok.Data;

import java.util.ArrayList;
import java.util.List;

@Data
public class Quai {
    private String id;
    private List<Retournement> retournements;

    public Quai(String id) {
        this.id = id;
        this.retournements = new ArrayList<>();
    }
}
